utils/api/benefits/checkAfs.ts

```diff
@@ -154,3 +159,28 @@ export default function checkAfs(params: CalculationInput): BenefitResult {
   // fallback
   throw new Error('should not be here')
 }
+
+class AfsEntitlement {
```

Comment:
Nice work keeping these classes closely related! If you wanted extra nerd points, you could define an interface / type that enforces this so the developers that touch this after us don't unknowingly blow it up. 

100% not required though!

Reply:
I am pretty content with how quickly it breaks when you do something wrong thanks to the `OutputItemAfs` type (and the types for GIS and Allowance elsewhere), despite not using a base class here.

Always happy to get nerd points but pretty happy with it as is! 😃